otel-tests/src/lib.rs

```diff
@@ -46,7 +52,7 @@ pub fn init_tracing() {
             // apply warning level globally
             .add_directive(LevelFilter::WARN.into())
             // log traces from test itself
-            .add_directive("happy_path=debug".parse().unwrap())
+            .add_directive(dbg!(format!("{module}=debug")).parse().unwrap())
```

Comment:
I guess you only used `dbg!` for debugging :) (clippy would complain anyway before merging)

```suggestion
            .add_directive(format!("{module}=debug").parse().unwrap())
```

Reply:
Good point! I missed this